crypto/armor/armor.go

```diff
@@ -5,7 +5,7 @@ import (
 	"fmt"
 	"io/ioutil"
 
-	"golang.org/x/crypto/openpgp/armor" // nolint: staticcheck
+	"golang.org/x/crypto/openpgp/armor" //nolint: staticcheck
```

Comment:
```suggestion
	"golang.org/x/crypto/openpgp/armor"
```

If a linter is giving us an issue over this, that's a bit silly and we should disable it imo.